### ISSUE TITLE
Need to fix the lat,lng estimates for labels currently in database

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2375]
My understanding is that due to the lack of GSV depth data, the labels we collected last week are not being shown on LabelMap, etc. because they have nulls for their lat,lng. So, we need to backfill these with lat,lng estimates. Given that this estimation algorithm is an open ticket (https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2374), I wonder what the best approach is here.

I think, in the short-term, we may just want to backfill these with a simple heuristic (e.g., with the pano's lat,lng). In the longer term and as we come up with a more robust/accurate approach in (https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2374), we'll want to use that.

Which also suggests that we need an additional column, perhaps, tracking how each label's lat,lng was estimated so we know which to update in the future.

**Comment by misaugstad** [comment]
I did actually do this already in #2367, and the old labels are visible again on labelmap. I think this is on me for not being super explicit in that PR description. But I added the new column that says which approximation method was used. And I back filled the table with the approximation that we use for the mini map. That approximation is assuming that you are placing a label 10 meters in front of where the camera is facing at the time when you place a label. So it uses the panorama's lat/lng and the camera heading to calculate a location. It will also be easy to update the old data when we make new approximations.

Closing via #2367
